test_outputs/snapshot_tests/cargo_semver_checks__snapshot_tests__configuration_conflict-output.snap

```diff
@@ -0,0 +1,11 @@
+---
+source: src/snapshot_tests.rs
+expression: result
+---
+--- error ---
+add a `priority` key to the entries in [package.metadata.cargo-semver-checks.lints].
```

Comment:
Not a great error message.

Imagine the user's `Cargo.toml` has 100 config entries. Suddenly, one of them caused a conflict. (Maybe it's because *we* added a new lint that is the first one to be in a particular combination of two groups simultaneously? The user had specified settings for each group, and they didn't conflict in our _previous_ release. In the new release with the new lint, they do! Oops!)

Now we just told the user "hey please add `priority` to all your 100 config entries." They'll uninstall the tool immediately.